engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -20,6 +20,7 @@
  *
  */
 
+#include "backends/platform/sdl/sdl-sys.h"
```

Comment:
Including the backend is not allowed, as we may have non-SDL backends. There is some limited use of such includes in the grim-engine, but those are themselves guarded with #if(SDL_BACKEND)

What's wrong with the values passed to initRenderer?

Reply:
Actually I had problems with the window size since the max resolution of my laptop is too small in the height dimension for Alpha Polaris. The code above fixed an issue with the mouse cursor, which first would not reach the bottom of the window, probably because the parameters above do not represent the size of the actual drawable area of the window but instead the resolution of the game (which seems fixed for wme games, although wme also seems to try hard to find an almost fitting resolution in case the preferred one is not available). However, I have just plugged in a monitor with a bigger resolution, so for the moment everything is fine anyways. I will remove the code from above.